RELEASE-NOTES.md

```diff
@@ -1,5 +1,13 @@
 # Wikibase DataModel release notes
 
+## Version 6.0.0 (dev)
+
+* Removed `Entity` class (deprecated since 1.0)
+* `Item` and `Property` no longer extend `Entity`
+    * Removed `getLabel`, `getLabels`, `getDescription`, `getDescriptions`, `getAliases`,
+      `getAllAliases`, `setLabels`, `setDescriptions`, `addAliases`, `setAllAliases`,
+      `removeLabel`, `removeDescription` and `removeAliases` methods
+
```

Comment:
Verified that these cover all the methods that used to be in Entity, and no longer are in Item and Property.